adafruit_stmpe610.py

```diff
@@ -320,11 +329,13 @@ def __init__(  # pylint: disable=too-many-arguments
         # Check device version.
         version = self.get_version
         if _STMPE_VERSION != version:
-            raise RuntimeError("Failed to find STMPE610! Chip Version 0x%x" % version)
+            raise RuntimeError(
+                f"Failed to find STMPE610! Chip Version {hex(version).upper()}."
+            )
         super().__init__()
 
     @property
-    def touch_point(self):  # pylint: disable=too-many-branches
+    def touch_point(self) -> Tuple[int, int, int]:  # pylint: disable=too-many-branches
```

Comment:
Since this can also return `None`, it should be `Optional[Tuple[int, int, int]]`.